web/src/assets/_site.scss

```diff
@@ -334,5 +334,41 @@ li.list-group-item:hover, {
 }
 
 .align-center {
-    text-align: center;
+  text-align: center;
+}
+  .nav.nav-tabs {
```

Comment:
This scss is repeated in tabs, the main css file is main.scss, site.scss is common sass file, but in order to divide and structure we have moved differnent component sass to it's files, in this case tabs.

Reply:
done